packages/admin-ui/ui/src/domain/inventory/locations/components/address-form/index.tsx

```diff
@@ -78,7 +78,9 @@ const AddressForm = ({
             required={addressFieldsRequired}
             {...register(path("address_1"), {
               pattern: FormValidator.whiteSpaceRule("Address 1"),
-              required: addressFieldsRequired,
+              required: addressFieldsRequired
+                ? "This field is required"
```

Comment:
nit(non-blocking): More specific error message

```suggestion
                ? FormValidator.required("Address 1")
```